tests/Application/UseCases/GetAllApproversCategoriesTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\PageApprovals\Tests\Application\UseCases;
+
+use PHPUnit\Framework\TestCase;
+use ProfessionalWiki\PageApprovals\Application\UseCases\GetAllApproversCategories;
+use ProfessionalWiki\PageApprovals\Adapters\UsersLookup;
+use ProfessionalWiki\PageApprovals\Adapters\DatabaseApproverRepository;
+use MediaWiki\Permissions\PermissionManager;
+use MediaWiki\MediaWikiServices;
+use User;
+
+/**
+ * @covers \ProfessionalWiki\PageApprovals\Application\UseCases\GetAllApproversCategories
+ * @group Database
+ */
+class GetAllApproversCategoriesTest extends TestCase {
```

Comment:
I did take ChatGPTs help to write this but it took several attempts and I pretty much asked ChatGPT to do exactly what I wanted.